src/current/v23.2/admission-control.md

```diff
@@ -34,7 +34,7 @@ This is particularly important for CockroachDB {{ site.data.products.serverless
 Admission control can help if your cluster has degraded performance due to the following types of node overload scenarios:
 
 - The node has more than 32 runnable goroutines per CPU, visible in the **Runnable goroutines per CPU** graph in the [**Overload** dashboard]({% link {{ page.version.version }}/ui-overload-dashboard.md %}#runnable-goroutines-per-cpu).
-- The node has a high number of files in level 0 of the Pebble LSM tree, visible in the **LSM L0 Health** graph in the [**Overload** dashboard]({% link {{ page.version.version }}/ui-overload-dashboard.md %}#lsm-l0-health).
+- The node has a high number of files in level 0 of the Pebble LSM tree, visible in the **IO Overload** graph in the [**Overload** dashboard]({% link {{ page.version.version }}/ui-overload-dashboard.md %}#io-overload).
```

Comment:
Maybe we can reword this to remove "high number of files in level 0" since that is not user visible anymore. 

We can instead say something along the lines of, "The node has a high amount of overload in the pebble LSM tree, visble in the **IO Overload** graph..."

Reply:
updated